content/zh-cn/privacy-policy/index.md

```diff
@@ -1,10 +1,35 @@
 ---
 title: Privacy Policy
-description: We do not use cookies and we do not collect any personal data.
+description: Privacy Policy and the update checker
 date: 2020-08-27T19:23:18+02:00
-lastmod: 2020-08-27T19:23:18+02:00
+lastmod: 2024-12-18T19:23:18+02:00
 draft: false
 images: []
 ---
 
-# Statistics collected by the update checker
+## Full disclosure: Statistics collected by the update checker
+
+We are aware that some of our users care a lot about their privacy.
+
+First, the short version: If you do not want to participate in anonymous data collection, do not enable the built-in update checker. If you want to know what information is collected, read on.
+
+Aegisub's update checker function retrieves a file describing any potential updates available for your installed version of Aegisub. Along with this request, some anonymous information about your operating environment are sent, to allow us as developers to better target our efforts, and to loosely gauge how large the Aegisub user base is.
+
+If you have enabled automatic update checking, the update checker runs automatically every time you start Aegisub, although at most once in a one-hour period. If updates are found, you are offered to postpone further checking for a week. If you choose to postpone further checks, it will be approximately 7 days until the next time a check is run. Aegisub never checks for updates when it is not running; there are no background processes.
+
+As of version 3.4.0, the following information is recorded when a check for updates is made:
+
+### Date and time
+Date and time are used to group the statistics by time periods and to see trends across time. For example, to track the adoption rate of a new version.
+
+### IP Address
+IP addresses are used to are aggregated into geographical regions to track where Aegisub is being used the most, to target localisation efforts, and possibly to select locations of download mirrors at a later point in time. IP addresses are also correlated to group all checks performed by a single user, to approximate the number of users we have. All of this is completely automated and we do not attempt to identify individuals, only count them anonymously as part of a statistics.
```

Comment:
```suggestion
IP addresses are stored and aggregated into geographical regions to track where Aegisub is being used the most, to target localisation efforts, and possibly to select locations of download mirrors at a later point in time. IP addresses are also correlated to group all checks performed by a single user, to approximate the number of users we have. All of this is completely automated and we do not attempt to identify individuals, only count them anonymously as part of a statistics.
```

I question whether "we do not attempt to identify individuals" will amount to anything in court, but IANAL.